tracer/src/Datadog.Trace/Span.cs

```diff
@@ -545,5 +548,13 @@ private void WriteCloseDebugMessage()
                 "Span closed: [s_id: {SpanId}, p_id: {ParentId}, t_id: {TraceId}] for (Service: {ServiceName}, Resource: {ResourceName}, Operation: {OperationName}, Tags: [{Tags}])\nDetails:{ToString}",
                 new object[] { Context.RawSpanId, Context.ParentIdInternal, Context.RawTraceId, ServiceName, ResourceName, OperationName, Tags, ToString() });
         }
+
+        internal SpanLink AddSpanLink(Span spanLinkToAdd, List<KeyValuePair<string, object>> optionalAttributes = null)
```

Comment:
nitpick: I think `optionalAttributes` could be renamed to `attributes` here

Reply:
implemented!